docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID for OpenSSH servers.
+---
+
+In this guide, you will learn how to create host certificates through Machine ID to allow Teleport users (manually or automated through Machine ID) to access OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived host certificates on Teleport clusters can be successfully applied.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+## Step 1/4. Download and install Teleport (=teleport.version=)
+
+In this step, you will be downloading and installing Teleport binaries onto the
+machine you wish to assign an identity to.
+
+Each Teleport package hosted on our downloads page ships with several useful
+binaries, including `teleport`, `tctl`, `tsh`, and `tbot`:
+
+- `teleport` is the daemon used to initialize a Teleport cluster; this binary is not used in this guide.
+- `tctl` is the administrative tool you will use to create the bot user.
+- `tsh` is the client tool you will use to log in to the OpenSSH server.
+- `tbot` is the Machine ID tool you will use to associate a bot user with a machine.
+
+Machine ID is available starting from the Teleport 9.0.0 release. Download the appropriate Teleport package for your platform from our [downloads page](https://goteleport.com/teleport/download).
+
+Next, log in to your cluster using `tsh` to ensure that your machine will be able to interact with the cluster:
```

Comment:
Referring to both the engineer's laptop and the machine on which we are installing `tbot` on as "machine" feels a little confusing - it might not be clear that this command needs to be run on the engineer's laptop.

Perhaps we ought to list all the machines involved at the start, explain what each one will be doing and give them a name we can bold when used in later instructions.